### PR TITLE
AEZ314 joinned the party!

### PR DESCRIPTION
**Pull request by AEZ314**
## Describe your project in 1-3 sentences

I am building a CO2 laser cutter. It's my first time stepping out of the "abstract side" of coding and getting physical. I'll complete this project along with a few friends --this will be their first experience.

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: AEZ314.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| CO2 Tube | https://www.abplaser.com/hy-dy-serisi-reci-guc-kaynagi?search=t%C3%BCp&description=true | $130  |
+| Stepper Motors | https://www.amazon.com.tr/dp/B07KZL4XCL/ref=asc_df_B07KZL4XCL1661961600000/?creative=380333&creativeASIN=B07KZL4XCL&linkCode=asn | $31.69 |
+| Lens/Mirror/Nozzle Set | https://www.amazon.com.tr/Cloudray-K4060-Grav%C3%BCr-Kesici-Kafas%C4%B1/dp/B0B5D26BRX/ref=sr_1_19?__mk_tr_TR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=9XMA67OD6O0Q&keywords=co2%2Blazer%2Bayna%2Blens%2Bset&qid=1671421766&sprefix=co2%2Blazer%2Bayna%2Blens%2Bse%2Caps%2C168&sr=8-19&th=1 | $84.51  |
+
+| Total           |                                       | $246.2 |
+
+There are some few other components, but I'll cover them myself as they exceed the $250 grant.
```
Comment (exu3) [inline AEZ314.md:47]: Do you think you could list the other components you'll need for your project? I think it'd make this project proposal feel more complete!

File: AEZ314.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| CO2 Tube | https://www.abplaser.com/hy-dy-serisi-reci-guc-kaynagi?search=t%C3%BCp&description=true | $130  |
+| Stepper Motors | https://www.amazon.com.tr/dp/B07KZL4XCL/ref=asc_df_B07KZL4XCL1661961600000/?creative=380333&creativeASIN=B07KZL4XCL&linkCode=asn | $31.69 |
+| Lens/Mirror/Nozzle Set | https://www.amazon.com.tr/Cloudray-K4060-Grav%C3%BCr-Kesici-Kafas%C4%B1/dp/B0B5D26BRX/ref=sr_1_19?__mk_tr_TR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=9XMA67OD6O0Q&keywords=co2%2Blazer%2Bayna%2Blens%2Bset&qid=1671421766&sprefix=co2%2Blazer%2Bayna%2Blens%2Bse%2Caps%2C168&sr=8-19&th=1 | $84.51  |
+
+| Total           |                                       | $246.2 |
```
Comment (exu3) [inline AEZ314.md:45]: Have you factored shipping costs / taxes in your total?

**Review by exu3 — CHANGES_REQUESTED** [review]
Hey @AEZ314! This project sounds really neat. Could you make the changes requested before we move forward? Thanks!

File: AEZ314.md (hunk 2 above)
Comment (AEZ314) [reply AEZ314.md:45]: Yes. The Taxes are included in the price, and the shipping is free for purchases exceeding a certain price threshold, which these items do.

File: AEZ314.md (hunk 1 above)
Comment (AEZ314) [reply AEZ314.md:47]: Will do!

Comment (AEZ314) [reply AEZ314.md]: Added the other parts :)

**Review by exu3 — APPROVED** [review]
Looks great! Thanks for making the changes. I'm excited to see how this turns out.

@MelSmith104 will follow up with next steps on getting onto Hack Club Bank and receiving your grant.

**Review by MelSmith104 — COMMENTED** [review]
hey @AEZ314 ! this looks so amazing! im super excited to meet you :) next step here is for us to hop on an onboarding call and get you set up on hack club bank! could you please let me know your slack handle and we'll go from there!

**Comment by AEZ314** [comment]
> hey @AEZ314 ! this looks so amazing! im super excited to meet you :) next step here is for us to hop on an onboarding call and get you set up on hack club bank! could you please let me know your slack handle and we'll go from there!
Hello @MelSmith104 ! So am I :D U04FS65LKR8 is my Slack user ID.  Should I prepare any documents whatsoever for the onboarding call? Also, I think our communication will continue on Slack, am I correct?

**Comment by MelSmith104** [comment]
hey @AEZ314 ! yep :) im going to dm you soon and we'll move communication over to slack. i can help you prep for your onboarding call there too!

**Pull request merged**